internal/view/pod.go

```diff
@@ -360,6 +364,24 @@ func podIsRunning(f dao.Factory, path string) bool {
 	return re.Phase(po) == render.Running
 }
 
+func getPodOS(f dao.Factory, path string) string {
```

Comment:
As mentioned above change the signature here to return string, error if unable to retrieve OS